test/iqlquery.test.ts

```diff
@@ -75,6 +75,7 @@ test("Local IQL Query", async () => {
     for (const e of table) {
         const queryRequest: ExecuteIQLQueryRequest = {
             query: {
+                "@type": "IQLQuery",
```

Comment:
Could this be a bug? Why do we need to add the IQLQuery as type here? (See type definition within the SDK above). From my point of view, it is not necessary within the ExecuteIQLQueryRequest to require a @ type definition?